app/controllers/main/routes.py

```diff
@@ -47,50 +49,134 @@ def events(selectedTerm=None):
-                               )
+        flash("Successfully banned user", "success")
+        return banUser(program_id, username, banNote, banEndDate, g.current_user)
+    except Exception as e:
+        print("Error  while updating ban", e)
+        return "", 500
+
+# ===========================Unban===============================================
+@main_bp.route('/<username>/unban/<program_id>', methods=['POST'])
+def unban(program_id, username):
+    """
+    This function updates the ban status of a username either when they are unbanned from a program.
+    program_id: the primary id of the program the student is being unbanned from
+    username: unique value of a user to correctly identify them
+    """
+    postData = request.form
+    unbanNote = postData["note"] # This contains the note left about the change
+    try:
+        flash("Successfully Unbanned user", "success")
+        return unbanUser(program_id, username, unbanNote, g.current_user)
+
+    except Exception as e:
+        print("Error  while updating Unban", e)
+        return "", 500
+
+
+@main_bp.route('/<username>/addInterest/<program_id>', methods=['POST'])
+def addInterest(program_id, username):
+    """
+    This function adds a program to the list of programs a user interested in
+    program_id: the primary id of the program the student is adding interest of
+    username: unique value of a user to correctly identify them
+    """
+    try:
+# <<<<<<< HEAD
+        return addUserInterest(program_id, username)
+
+# =======
+#         profileUser = User.get(User.username == username)
+#         upcomingEvents = getUpcomingEventsForUser(username)
+#         programs = Program.select()
+#         interests = Interest.select().where(Interest.user == profileUser)
+#         interests_ids = [interest.program.id for interest in interests]
+#         rsvpedEventsList = EventRsvp.select().where(EventRsvp.user == profileUser)
+#         rsvpedEvents = [event.event.id for event in rsvpedEventsList]
+#
+#         allUserEntries = list(BackgroundCheck.select().where(BackgroundCheck.user == profileUser))
+#         completedBackgroundCheck = {entry.type.id: entry.passBackgroundCheck for entry in allUserEntries}
+#         backgroundTypes = list(BackgroundCheckType.select())
+#
+#
+#         return render_template('/volunteer/volunteerProfile.html',
+#                                title="Volunteer Interest",
+#                                user = profileUser,
+#                                programs = programs,
+#                                interests = interests,
+#                                interests_ids = interests_ids,
+#                                upcomingEvents = upcomingEvents,
+#                                rsvpedEvents = rsvpedEvents,
+#                                backgroundTypes = backgroundTypes,
+#                                completedBackgroundCheck = completedBackgroundCheck
+#                                )
+# >>>>>>> development
```

Comment:
Get rid of the merge artifacts once you have resolved them, don't just comment them out